page-optimize.php

```diff
@@ -277,6 +277,11 @@ function page_optimize_bail() {
 		return true;
 	}
 
+	// Bail if we're editing pages in Brizy Editor
+	if ( class_exists( 'Brizy_Editor' ) && ( isset( $_GET[ Brizy_Editor::prefix( '-edit-iframe' ) ] ) || isset( $_GET[ Brizy_Editor::prefix( '-edit' ) ] ) ) ) {
```

Comment:
Maybe we should also confirm that `Brizy_Editor::prefix()` exists. That way, if `Brizy_Editor::prefix()` goes away, we only break Brizy and only when using the async load mode.

Reply:
Nice idea. Will add.